spec/base.yml

```diff
@@ -23,3 +23,14 @@ tags:
     name: Data
   - description: Extension to OGC API - Features to support STAC metadata model and search API
     name: STAC
+  - description: >-
+      All write requests require authentication. There are several options for a user to
+      authenticate himself:
+
+      * When using the browsable API the user can simply use the admin interface for logging in
+
+      * Using basic authentication for the write requests.
```

Comment:
Pls add this to `transaction/transaction.yml`, not `base.yml` since it only concerns the transactional part.

Note that you have to repeat all tags, so you need  to add (uncomment):
```
tags:
  - description: Essential characteristics of this API
    name: Capabilities
  - description: Access to data (features)
    name: Data
  - description: Extension to OGC API - Features to support STAC metadata model and search API
    name: STAC
  - name: 'Data Management'
    description: >-
      STAC-specific operations to add, remove, and edit items within OGC API - Features
      collections.
```

Yeah, curl examples would be nice, either here or in one of the Markdown files.

Reply:
Makes sense, moved it